x-pack/legacy/plugins/siem/server/lib/detection_engine/signals_mapping.json

```diff
@@ -1592,50 +1860,50 @@
           "device": {
             "properties": {
               "name": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               }
             }
           },
           "name": {
-            "type": "keyword",
-            "ignore_above": 1024
+            "ignore_above": 1024,
+            "type": "keyword"
           },
           "original": {
-            "type": "keyword",
-            "ignore_above": 1024
+            "ignore_above": 1024,
+            "type": "keyword"
           },
           "os": {
             "properties": {
               "family": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               },
               "full": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               },
               "kernel": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               },
               "name": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               },
               "platform": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               },
               "version": {
-                "type": "keyword",
-                "ignore_above": 1024
+                "ignore_above": 1024,
+                "type": "keyword"
               }
             }
           },
           "version": {
-            "type": "keyword",
-            "ignore_above": 1024
+            "ignore_above": 1024,
+            "type": "keyword"
           }
         }
       }
```

Comment:
Sorry for the length of this file. When I moved over to what was checked in the generated folder for ECS 1.2, it outputs the `type` and `ignore` in a different order but I think it's better to match almost exactly what they have with the ECS 1.2 generated.